packages/components/src/Tab.tsx

```diff
@@ -33,12 +34,19 @@ StyledLi.defaultProps = {
   },
 };
 
+export const Count = styled.span`
+  font-weight: 400;
+`;
+
 export class Tab extends React.Component<TabProps> {
   public render(): JSX.Element {
     const TabComponent = this.props.TabComponent || StyledLi;
+    const hasCount = this.props.tabCount ? <Count>{this.props.tabCount}</Count> : "";
+
     return (
       <TabComponent isActive={this.props.isActive} onClick={this.onClick}>
         {this.props.title}
```

Comment:
Instead of supporting a `tabCount` prop, should we just let `title` be either a string or a JSX.Element? 

This way the implementer has control over what gets rendered and we don't have to worry about supporting another property for this component.